src/learner.Rmd

```diff
@@ -90,7 +90,7 @@ It contains, among others, the type of hyperparameters (e.g., numeric, logical),
 default values and the range of allowed values.
 
 Moreover, [%mlr] provides function [&getHyperPars] to access the current hyperparameter setting
-of a [Learner](&makeLearner) and [&getParamSet] to get a description of all possible settings.
+of a [Learner](&makeLearner), [&getParamSet] or [&getLearnerParamSet] to get a description of all possible settings and [&getLearnerParVals] to get the configured hyperparameters.
```

Comment:
Could you please make more clear that
* getLearnerParVals is an alias for getHyperPars and
* getLearnerParamSet is an alias for getParamSet?